komodo/release_transpiler.py

```diff
@@ -127,6 +205,13 @@ def main():
         required=True,
         help="Folder to output new release files",
     )
+    transpile_parser.add_argument(
+        "--matrix",
+        help="Matrix to be transpiled.",
```

Comment:
> And to be honest I think it would be ok once we specify matrix-coordinates.

sure, sure - we can skip renaming the first one. don't wanna upset all the users of komodo out there : )

> But I did not play with it

i tried it out, sadly it doesn't display the default value - if the arg parse lib doesn't support it, you could of couse just pull out the default value into a variable...

Reply:
nice find! i'm still missing that it expects yaml formatted input in the help text. although i don't know how much it will help, and the default provides a working example, and makes it obvious how to extend it.

oh one point though that i forgot to mention before, you can remove the quotes around `rhel` and `python` in the default value, according to my testing at least

Reply:
perfect, thank you!